src/content/docs/extensions/json.mdx

```diff
@@ -548,34 +548,44 @@ Output:
 ```
 
 ```cypher
-RETURN to_json({duck : 42});
+RETURN to_json({kuzu : 42});
 ```
 Output:
 ```json
-{"duck":42}
+{"kuzu":42}
 ```
 
 ```cypher
-RETURN to_json(map(['duck'],[42]));
+RETURN to_json(map(['kuzu'],[42]));
```

Comment:
Same comment - it's better to use a more realistic example from above. Also, why are both the inputs to the `map` function lists?